routes.js

```diff
@@ -135,6 +135,7 @@ const sharedRoutes = {
 
       return {
         title: isPR ? `Pull Request #${number}` : `Issue #${number}`,
+        headerLeft: navigation.state.params.headerLeft,
```

Comment:
You're right, I haven't tried to swipe left, and it still allow the user to navigate back... 

(-> Comment 2)